cyclus_input/predicting_the_past.xml

```diff
@@ -13,7 +13,7 @@
     <spec><lib>cycamore</lib><name>Source</name></spec>
     <spec><lib>cycamore</lib><name>Storage</name></spec>
   </archetypes>
-  <xi:include href="./cyclus_input/recipes/lifetimes.xml#xpointer(/simulation/facilities/child::*)" />
+  <xi:include href="./recipes/lifetimes.xml#xpointer(/simulation/facilities/child::*)" />
```

Comment:
if you want to include the whole lifetimes.xml file, I think you can use   

```xml
<xi:include href="./recipes/lifetimes.xml" xpointer="xpointer(*/*)" />  
```
Have you tried something like that?